packages/main/pyproject.toml

```diff
@@ -82,7 +82,7 @@ htmldocx = "^0.0.6"
 hubspot-api-client = "^4.0.6"
 pyotp = "^2.6.0"
 importlib-metadata = "^4.13.0"
-O365 = "<=2.0.17"
+O365 = "^2.0.17"
```

Comment:
if tzlocal 4.2 works along with the rpaframework then O365 could be directly pinned into to latest 2.0.21

Reply:
Yup, good point.